clickable

```diff
@@ -1093,7 +1099,35 @@ class PythonClickable(PureClickable):
     # The only difference between this and the Pure template is that this doesn't force the "all" arch
     pass
 
+class GoClickable(Clickable):
+    def go_install(self):
+        get_output=True
+        if not os.path.exists(self.temp):
+            os.makedirs(self.temp)
+        gocommand = "/usr/local/go/bin/go build -i -o {}/{} .".format(self.temp,
+                                                                    self.config.app)
+        wrapped_command = 'docker run -v {}:{} -v {}:/home/developer/gopath -w {} -u {} --rm -i {} bash -c "{}"'.format(
```

Comment:
You should use run_container_command here, so you don't need to do all the docker setup

Reply:
For go I need to mount the gopath and the run_container_command is only able to mount one volume

Reply:
Also I need to fix that the go compile doesn't need root.

Reply:
@nanu-c you could update the run_container_command function to add that volume if a go template is being used.

Reply:
I did it, only problem, still need user 0